packages/components/addon/components/hds/copy/button/index.hbs

```diff
@@ -0,0 +1,20 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+<button
+  type="button"
+  class="{{this.classNames}} {{if this.isSuccess 'is-success'}}"
```

Comment:
Since the main class name for the button is added in the js file, it's probably a good idea to keep the logic for adding the 'is-success' class in the js together with it.

Also, wouldn't we want to namespace the classname as: `'hds-copy-button--is-success'`?